apps/server/src/modules/files-storage/helper/promise.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { getResolvedValues, isFulfilled } from './promise';
+
+describe('Promise Helper', () => {
+	describe('isFullFilled is called', () => {
+		describe('promise is fullfilled', () => {
+			const setup = async () => {
+				const promise = Promise.resolve('value');
+				const promises = await Promise.allSettled([promise]);
+				const settledPromise = promises[0];
+
+				return settledPromise;
+			};
+
+			it('should return true', async () => {
+				const settledPromise = await setup();
+
+				const result = isFulfilled(settledPromise);
+
+				expect(result).toBe(true);
+			});
+		});
+
+		describe('promise is rejected', () => {
+			const setup = async () => {
+				const promise = Promise.reject();
+				const promises = await Promise.allSettled([promise]);
+				const settledPromise = promises[0];
+
+				return settledPromise;
+			};
+
+			it('should return false', async () => {
+				const settledPromise = await setup();
+
+				const result = isFulfilled(settledPromise);
+
+				expect(result).toBe(false);
+			});
+		});
+	});
+
+	describe('getResolvedValues is called', () => {
+		describe('receives fullfilled promises only', () => {
+			const setup = async () => {
+				const value1 = 'value1';
+				const value2 = 'value2';
+				const promise1 = Promise.resolve(value1);
+				const promise2 = Promise.resolve(value2);
+				const promises = await Promise.allSettled([promise1, promise2]);
+
+				return { promises, value1, value2 };
+			};
+
+			it('should return value1 and value2', async () => {
+				const { promises, value1, value2 } = await setup();
+
+				const result = getResolvedValues(promises);
+
+				expect(result).toEqual([value1, value2]);
+			});
+		});
+
+		describe('receives fullfilled and rejected promises', () => {
+			const setup = async () => {
+				const value2 = 'value2';
+				const promise1 = Promise.reject();
+				const promise2 = Promise.resolve(value2);
+				const promises = await Promise.allSettled([promise1, promise2]);
+
+				return { promises, value2 };
+			};
+
+			it('should return array with value2', async () => {
+				const { promises, value2 } = await setup();
+
+				const result = getResolvedValues(promises);
+
+				expect(result).toEqual([value2]);
+			});
+		});
+
+		describe('receives rejected promises only', () => {
+			const setup = async () => {
+				const promise1 = Promise.reject();
+				const promises = await Promise.allSettled([promise1]);
+
+				return { promises };
+			};
+
+			it('should return empty array', async () => {
```

Comment:
The same problem as mentioned above: "Promise Helper getResolvedValues is called receives rejected promises only should return empty array" is not a reasonable sentence. I would suggest the following structure:
```suggestion
			describe('Promise Helper', ...
			  describe('getResolvedValues' ,...
			    it('called with rejected promises only should return empty array', ...
```
This would compose to a reasonable sentence and would be coherent with Jest docs: https://jestjs.io/docs/api#describename-fn

Or as an alternative: `it('should return empty array if called with rejected promises only'`

Reply:
See comments above.